framework/source/class/qx/bom/client/EcmaScript.js

```diff
@@ -117,6 +117,26 @@ qx.Bootstrap.define("qx.bom.client.EcmaScript",
 
 
     /**
+     * Checks if 'find' is supported on the Array object.
+     * @internal
+     * @return {Boolean} <code>true</code>, if the method is available.
+     */
+    getArrayFind : function() {
```

Comment:
Yes, existing code trumps preferred naming. (Maybe at some point we fix all of 'em, but that's not part of this PR.)